spec/openapi.yaml

```diff
@@ -5250,6 +5485,193 @@ paths:
 # An object to hold reusable parts that can be used across the spec
 components:
   schemas:
+    chronosscheduleresponse:
+      title: chronosscheduleresponse
+      type: object
+      required:
+        - page_token
+        - page_size
+      properties:
+        page_token:
+          type: string
+          example: abc123
+        page_size:
+          type: number
+          example: 20
+    scheduleddetailreport:
+      title: scheduleddetailreport
+      type: object
+      required:
+        - label
+        - schedule
+      properties:
+        label:
+          type: string
+        schedule:
+          $ref: '#/components/schemas/schedule'
+        report:
+          $ref: '#/components/schemas/scheduleddetailrequest'
+    scheduleddetailrequest:
+      title: scheduleddetailrequest
+      type: object
+      required:
+        - period
+      properties:
+        period:
+          $ref: '#/components/schemas/ReportPeriod'
+        timezone:
+          type: string
+          example: Australia/Sydney
+          description: The standard timezone name
+        direction:
+          $ref: '#/components/schemas/MessageDirection'
+        source:
+          type: string
+          example: +61555555555
+        destination:
+          type: string
+          example: +61555555555
+        metadata_key:
+          type: string
+          example: broadcastId
+        metadata_value:
+          type: string
+          example: ABC
+        accounts:
+          type: array
+          items:
+            type: string
+        status:
+          type: array
+          items:
+            $ref: '#/components/schemas/MessageStatus'
+        opt_out:
+          type: string
+          example: true
```

Comment:
You might want to double-check this. We found that booleans and numbers are seen as just that even in these cases where `type` is set to`string`. As an example:
```
opt_out:
    type: string
    example: true
source:
    type: string
    example: +642177889955
```
would become
```
"source": 642177889955,
"opt_out": true,
```
rather than the actual desired
```
"source": "+642177889955",
"opt_out": "true",
```
To fix this, [we wrapped the potentially misinterpretable raw examples into a string](https://github.com/messagemedia/documentation/pull/30/files#diff-dfce4d9d11a31be72739d767812e1b3435a06045c19be411a28b517d2c208001R4905-R4920) which resolved the issue
```
opt_out:
    type: string
    example: "true"
source:
    type: string
    example: "+642177889955"
```
Definitely worth running the localhost Redoc to see how this behaves to make sure you get the expected outcome in these ambiguous cases.